src/Components/PasteBoilerplateTextContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useCallback, useState } from "react";
+
+export const PasteBoilerplateTextContext = createContext();
+
+export const PasteBoilerplateTextProvider = ({ children }) => {
```

Comment:
This looks like a copy of the `PasteBoilerplateContentPopoutContext.js `, do we still need this?